src/include/btmem.h

```diff
@@ -669,6 +672,19 @@ struct __wt_page {
 
     uint8_t unused[2]; /* Unused padding */
 
+    size_t memory_footprint; /* Memory attached to the page */
+
+    /* Page's on-disk representation: NULL for pages created in memory. */
+    const WT_PAGE_HEADER *dsk;
+
+    /* If/when the page is modified, we need lots more information. */
+    WT_PAGE_MODIFY *modify;
+
+    /*
+     * !!!
+     * This is the 64 byte boundary, try to keep hot fields above here.
+     */
```

Comment:
FYI this moves `read_gen` below this comment and before it was above it so considered a `hot field`. Is it no longer hot? Is that a possible performance issue?

Reply:
This is a real change, and is a potential performance issue.